### ISSUE TITLE
Disable automatic tracing for command

### ISSUE THREAD

**Issue by ptondereau** [auxmoney/OpentracingBundle-core#83]
I have a use case when my Symfony command are some kinds of worker.
In this context, I want to avoid waiting for my command to finish and only trace the part I want to.
WDYT?

**Comment by cawolf** [comment]
Hi @ptondereau ,
we also have this use case and kind of worked around it by just ignoring the warpping command traces and just ensuring, that the "inner" spans (which are linked to other parent traces from different services) are flushed to the tracer. We are using the internal Auxmoney\OpentracingBundle\Internal\Opentracing interface to get access to the OpenTracing\Tracer instance and flush after each span manually.

However, we can imagine having this as a "regular" feature, maybe by decorating a long running command with a marker interface and querying for this interface in the appropriate event subscribers.

Currently, we are focusing primarily on upgrading the bundle, and are looking for help with implementing new features. Would you yourself like to take a shot? If not, any help is welcome. :)

**Comment by ujwaldhakal** [comment]
are these handle by [this File](https://github.com/auxmoney/OpentracingBundle-core/blob/master/EventListener/StartCommandSpanSubscriber.php) ? 
For me I had to listen to these event & finish the active span from my symfony code

**Comment by cawolf** [comment]
Yes, @ujwaldhakal , this subscriber opens the root span for each Command. The [FinishCommandSpanSubscriber](https://github.com/auxmoney/OpentracingBundle-core/blob/master/EventListener/FinishCommandSpanSubscriber.php) is its counterpart.

**Comment by pronata** [comment]
What if add in subscribers make some filter function by command name not to begin trace automatically?

**Comment by pronata** [comment]
Hi @cawolf 

What if add in subscribers some filter function by command name not to begin trace automatically?

**Comment by pronata** [comment]
I found the solution through decorators (decorating StartCommandSpanSubscriber and FinishCommandSpanSubscriber by adding exclusions there).
